### ISSUE TITLE
[Feature request]: Device settings page needs a visual overhaul

### ISSUE THREAD

**Issue by RubenKelevra** [Koenkk/zigbee2mqtt#12652]
### Is your feature request related to a problem? Please describe

I find it pretty hard to parse the visuals of the device settings page (and other settings pages.

There's no rime or reason behind the boxes for values and the text around it - at least from the visual appearance:

![Screenshot_20220530_225021-1](https://user-images.githubusercontent.com/614929/171060275-ab7a8d8b-da84-4e43-8866-a0268c94e881.png)

### Describe the solution you'd like

Would be neat to have something like alternating background colors or other clear visual guides to separate the individual menu entries.

### Describe alternatives you've considered

None

### Additional context

None

**Comment by sjorge** [comment]
You probably want to file that here https://github.com/nurikk/zigbee2mqtt-frontend/issues

**Comment by Koenkk** [comment]
Please create issues related to the frontend here; https://github.com/nurikk/zigbee2mqtt-frontend/issues
